### DISCUSSION TITLE
Lista fatture elettroniche da registrare

### DISCUSSION THREAD

**Discussion by Albw88** [fattureincloud/api#214] [Q&A] [answered]
Non trovo la chiamate per ottenere le fatture elettroniche passive da registrare mi potete aiutare?

**Comment by emajo** [comment, marked as answer]
Ciao @Albw88, purtroppo non abbiamo un metodo per ottenere/registrare le fatture elettroniche passive.

Emanuele.

**Comment by aster94** [comment]
+1
sarebbe utile poterle gestire tramite api, se non sbaglio purtroppo non è nella roadmap, corretto?
